backends/qualcomm/quantizer/quantizer.py

```diff
@@ -182,7 +181,6 @@ def set_per_channel_linear_quant(self, enable: bool) -> None:
         self._update_per_channel_weight_quant_ops(linear_ops, enable)
 
     def transform_for_annotation(self, model: GraphModule) -> GraphModule:
-        model = RemoveRedundancy()(model).graph_module
```

Comment:
Should we have a follow up to add this back? Feel like we may have perf regresssion without it

Reply:
Thanks for reviewing. Although the pass RemoveRedundancy() is removed during quantizer, it will still be called during capture_program(), so the final performance should be the same.
[[](url)](https://github.com/pytorch/executorch/blob/7f6a3416c4e183a57f023969004afec74a4ea480/backends/qualcomm/utils/utils.py#L196)